pkg/codegen/nodejs/gen_program_utils.go

```diff
@@ -94,3 +75,40 @@ func getHelperMethodIfNeeded(functionName string, indent string) (string, bool)
 		return "", false
 	}
 }
+
+// During code generation, it's possible that we'll generate expression code that is "too safe" as arguments to try.
+// E.g. given some PCL code `try(a.b.c, "fallback")`, where perhaps `a.b` is not defined, we'd ideally generate
+// `a.b.c` in TypeScript, which will throw and hit the `catch` block. However, depending on the inferred optionality
+// of the expressions involved, we may generate e.g. `a?.b?.c`, which instead of throwing will return `undefined`.
+// We thus check for this explicitly in our helper. This should be safe, since `undefined` is _not_ strictly equal
+// to `null`, and `null` is the "official no-value" value for PCL.
+func generateTryFunction(outputTry bool, functionArgs []model.Expression, indent string) (string, bool) {
```

Comment:
* `functionArgs` looks unused here?
* This doc comment should probably be in the function above the code we generate.
* As a result, we probably want a doc comment for this function.